astro_2.0/features/CreateDao/components/SubjectRule/SubjectRule.module.scss

```diff
@@ -34,6 +34,10 @@
     flex-wrap: nowrap;
   }
 
+  > div {
+    margin-bottom: 16px;
+  }
+
   > div:not(:last-of-type) {
     margin-right: 16px;
   }
```

Comment:
It's quite "dangerous" to tighten styles to the structure. Maybe, we can achieve same by providing classNames to nested components?